posydon/popsyn/normalized_pop_mass.py

```diff
@@ -65,28 +66,28 @@ def imf_part_2(m, m_1, m_min, alpha1, alpha2):
     def imf_part_3(m, m_1, m_2, m_min, alpha1, alpha2, alpha3):
         return ((m_1/m_min)**-alpha1)*((m_2/m_1)**-alpha2)*((m/m_2)**-alpha3)
 
-    def mean_mass_of_binary(f0, f_bin, m_1, m_2, m_min, m_max,
+    def mean_mass_of_binary(f0, f_bin_nature, m_1, m_2, m_min, m_max,
                             alpha1, alpha2, alpha3):
         a1, a2, a3 = alpha1, alpha2, alpha3
-        mean_mass = f0 * (1 + (f_bin/2)) * (
+        mean_mass = f0 * (1 + (f_bin_nature/2)) * (
             (1/(2-a1)) * ((m_1**(2-a1) - m_min**(2-a1))/(m_min**-a1))
             + ((1/(2-a2))
                * (m_1/m_min)**-a1 * ((m_2**(2-a2) - m_1**(2-a2))/(m_1**-a2)))
             + ((1/(2-a3)) * (m_1/m_min)**-a1 * (m_2/m_1)**-a2
                * ((m_max**(2-a3) - m_2**(2-a3))/(m_2**-a3))))
         return mean_mass
 
-    def mean_mass_simulated(alpha3, m_a, m_b):
+    def mean_mass_simulated(alpha3, m_a, m_b, f_bin_simulated):
```

Comment:
Do we need this function? Wouldn't it be better to get the mean_mass_simulated=mass_simulated/count_simulated?

Reply:
In my opinion, it's just a different approach. However, you might be right—it does seem much more straightforward to use this.